doc/flame_fire_atlas/fire_atlas.md

```diff
@@ -0,0 +1,98 @@
+# FireAtlas
+
+FireAtlas is a tool for handling `texture atlases`. Using FireAtlas we can access images using keys
+from texture atlas like accessing key-value pair.
+
+
+## Texture atlas
+
+A texture atlas(also called a spritesheet or an image sprite in 2d game development) is an image
```

Comment:
```suggestion
A texture atlas is an image
```
image sprite is not the same thing as a texture atlas and I don't think you have to mention spritesheets here since it can be a subset what one means when saying texture atlas.